build/blocks/newsletter-form/src/script.js--output-path.js

```diff
@@ -0,0 +1 @@
),Object(c.createElement)("div",{class:"cata-block-streaming-guide-newsletter__form-placeholder"},Object(c.createElement)("div",{class:"cata-block-streaming-guide-newsletter__fieldset"},Object(c.createElement)("label",{for:"mce-EMAIL",class:"screen-reader-text"},"Enter Your Email"),Object(c.createElement)("input",{type:"email",value:"",name:"EMAIL",id:"mce-EMAIL",placeholder:"Enter Your Email",required:!0}),Object(c.createElement)("button",{class:"button is-primary is-filled",type:"submit"},"Subscribe"))),Object(c.createElement)(o.RichText,{tagName:"p",value:e.attributes.legalText,onChange:t=>e.setAttributes({legalText:t}),className:"cata-block-streaming-guide-newsletter__details",placeholder:"Unsubscribe at any time. By subscribing, you agree to the terms of our %%Privacy Policy%%",keepPlaceholderOnFocus:!0}))))))})),save:function(e){const t=o.useBlockProps.save();return Object(c.createElement)("div",t,Object(c.createElement)("div",{class:"cata-block-streaming-guide-newsletter",style:{position:"relative"}},Object(c.createElement)("img",{class:"cata-block-streaming-guide__bg-img",alt:e.attributes.mediaAltText,src:e.attributes.mediaUrl,style:{position:"absolute",zIndex:-1,width:"100%",height:"100%",objectFit:"cover",overflow:"hidden",display:!e.attributes.mediaUrl&&"none"}}),Object(c.createElement)("div",{class:"cata-block-streaming-guide-newsletter__inner"},Object(c.createElement)("div",{class:"cata-block-streaming-guide-newsletter__layout"},Object(c.createElement)("div",{class:"cata-block-streaming-guide-newsletter__start"},Object(c.createElement)("h3",{class:"cata-block-streaming-guide-newsletter__title"},e.attributes.title)),Object(c.createElement)("div",{class:"cata-block-streaming-guide-newsletter__end"},Object(c.createElement)("p",{class:"cata-block-streaming-guide-newsletter__description"},e.attributes.description),Object(c.createElement)("form",{action:""+e.attributes.mailchimpUrlBase+e.attributes.mailchimpUserId+e.attributes.mailchimpUrlAmp+e.attributes.mailchimpAudienceId,method:"post",name:"mc-embedded-subscribe-form",target:"_blank",validate:!0,"data-mailchimp":!0},Object(c.createElement)("div",{class:"cata-block-streaming-guide-newsletter__response",role:"alert"}),Object(c.createElement)("fieldset",{class:"cata-block-streaming-guide-newsletter__fieldset"},Object(c.createElement)("label",{for:"mce-EMAIL",class:"screen-reader-text"},"Enter Your Email"),Object(c.createElement)("input",{type:"email",value:"",name:"EMAIL",id:"mce-EMAIL",placeholder:"Enter Your Email",required:!0}),Object(c.createElement)("button",{class:"button is-primary is-filled",type:"submit"},"Subscribe"),Object(c.createElement)("div",{style:"position: absolute; left: -5000px;","aria-hidden":"true"},Object(c.createElement)("input",{type:"text",name:"b_"+e.attributes.mailchimpUserId+"_"+e.attributes.mailchimpAudienceId,tabindex:"-1",value:""})))),Object(c.createElement)("p",{class:"cata-block-streaming-guide-newsletter__details"},e.attributes.legalText))))))}})}])}});
```

Comment:
I think this file is here by mistake. It has `--output-path.js` as the end of the filename.